client/landing/stepper/declarative-flow/internals/steps-repository/components/accent-color-control/style.scss

```diff
@@ -0,0 +1,54 @@
+@import "../../style";
+
+.accent-color-control__accent-color-input {
+	background-size: 25px, auto;
+	background-position: left 13px center, 95% !important;
+	background-repeat: no-repeat, no-repeat !important;
+	padding-left: 50px !important;
+
+	/* Overwrites the transition value set by FormInput component.
+    It was the cause of the https://github.com/Automattic/wp-calypso/issues/67326 */
+	transition: none;
+	cursor: pointer;
+}
+
+.accent-color-control__popover {
+	.popover__arrow {
+		display: none;
+	}
+	&:focus,
+	&:focus-visible {
+		outline: none;
+	}
+}
+
+.accent-color-control__contrast-warning {
+	display: flex;
+	font-size: $font-body-small;
+	height: 0;
+	transition: height 0.5s ease-in-out, opacity 0.5s ease-in-out;
+	opacity: 0;
+	margin-top: 16px;
+	margin-bottom: -16px;
+	overflow: hidden;
+
+	&.is-visible {
+		height: 80px;
+		opacity: 1;
+	}
+
+	.accent-color-control__contrast-warning-icon-container {
+		margin-top: 4px;
+		margin-right: 4px;
+
+		svg {
+			stroke: var(--studio-yellow-40);
```

Comment:
Hey! For my own understanding, I'm curious if there's a particular reason we're using CSS custom properties since we're working within Sass world anyway? Especially as they'll bloat our CSS that's sent to the browser, whereas Sass variables won't end up in the compiled CSS. 🙂

Reply:
@kirahsapong - In the case of this PR, I'm just moving code without changes, so there wasn't a decision to use this approach (or not). In general, you'll find a range of styles/approaches in the calypso codebase to JavaScript, TypeScript, React, components, api calls, CSS, etc. I think that reflects the fact that code base has been around a long time through many code 'eras', and because of its size, can be tricky to refactor for consistency.

With respect to CSS properties, Calypso has made extensive use of CSS custom properties. See [this file](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-color-schemes/src/shared/color-schemes/_default.scss) and the associated directory. But again, you'll see other approaches used elsewhere. 

If you haven't already, you should peruse the [Calypso CSS guidelines](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/css.md) for up-to-date standards on CSS, though I don't think there's much there about custom properties one way or the other.